src/eventServer/eventHandlers/notificationEvent.ts

```diff
@@ -53,11 +55,18 @@ const onEvent: ConsumeCallback<NotificationMessage | Array<NotificationMessage>>
   payload.filter(m => m.value.metadata.event.action === 'commit')
   .forEach(message => {
     // Pretend this is related to a pre-specified thirdpartyRequest/transaction
-    temporaryMockTransactionCallback(config.MOCK_CALLBACK, message)
+    const mockThirdpartyTransactionRequest = temporaryMockTransactionCallback(config.MOCK_CALLBACK, message)
     // console.log("TODO: sending callback to PISP", mockThirdpartyTransactionRequest)
 
-    // TODO - handle this in domain, and send request to the PISP!
-    // handled in https://app.zenhub.com/workspaces/mojaloop-project-59edee71d1407922110cf083/issues/mojaloop/mojaloop/270
+    // Enum.EndPoints.FspEndpointTemplates.TP_TRANSACTION_REQUEST_POST is a temporary template.
+    // todo: switch to a patch template endpoint after it's checked in
+    forwardTransactionRequestNotification(
+      mockThirdpartyTransactionRequest.value.content.headers as Hapi.Util.Dictionary<string>,
+      mockThirdpartyTransactionRequest.value.id,
+      mockThirdpartyTransactionRequest.value.content.payload,
+      Enum.EndPoints.FspEndpointTemplates.TP_TRANSACTION_REQUEST_POST,
```

Comment:
Hmm I'm not sure what we are doing elsewhere, but I don't like that the endpoint templates and endpointTypes are specified in different places, that seems to me like we aren't separating concerns properly.

Either we have them both in the domain function, or both here - what do you think?

Reply:
glad you thought it was strange, since it's done in the other domain functions atm.
lets move them here. it abstracts notification forwarding better if there is a case for another type of `onEvent` request other than commit -> PATCH thirdparty

Reply:
cool

Reply:
In case of transaction and authorizations , we are using same domain function to handle POST and PUT endpoints. so either we have to pass  FspEndpointType and FspEndpointTemplate values from handlers or add condition to check method type.

Reply:
Ah ok - I see now @sridharvoruganti